src/LiveSplit.Core/Server/CommandServer.cs

```diff
@@ -291,6 +296,58 @@ private void ProcessMessage(string message, IConnection clientConnection)
                 State.IsGameTimePaused = true;
                 break;
             }
+            case "getgamename":
+            {
+                response = State.Run.GameName.ToString();
+                break;
+            }
+            case "getcategoryname":
+            {
+                response = State.Run.CategoryName.ToString();
+                break;
+            }
+            case "getcategoryvariables":
+            {
+                RunMetadata md = State.Run.Metadata;
+
+                //Region
+                string region = null;
+                if (md.Region != null && !string.IsNullOrEmpty(md.Region.Abbreviation) && md.Game != null)
```

Comment:
Any particular reason as to why you don't want to accept my pattern matching suggestions here?
```suggestion
                if (md is { Game: not null, Region.Length: > 0 })
```